src/mono/netcore/sample/Android/AndroidSampleApp.csproj

```diff
@@ -32,6 +32,21 @@
 
     <RemoveDir Directories="$(ApkDir)" />
 
+    <!-- TEMP: consume OpenSSL binaries from external sources via env. variables.-->
```

Comment:
I'm curious: does the sample not work if we don't have these?

Reply:
I added this change because I had a crash like below:
```
--------- beginning of crash
11-24 17:09:04.896 14128 14128 E AndroidRuntime: FATAL EXCEPTION: main
11-24 17:09:04.896 14128 14128 E AndroidRuntime: Process: net.dot.HelloAndroid, PID: 14128
11-24 17:09:04.896 14128 14128 E AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "OCSP_RESPONSE_new" referenced by "/data/app/net.dot.HelloAndroid-flRYpqK33QVhtcZL5R0ivQ==/lib/x86/libSystem.Security.Cryptography.Native.OpenSsl.so"...
```